fem/quadinterpolator_face.cpp

```diff
@@ -355,6 +353,184 @@ void FaceQuadratureInterpolator::Eval3D(
    });
 }
 
+template<const int T_VDIM, const int T_ND1D, const int T_NQ1D>
+void FaceQuadratureInterpolator::SmemEval3D(
+   const int NF,
+   const int vdim,
+   const DofToQuad &maps,
+   const Array<bool> &signs,
+   const Vector &e_vec,
+   Vector &q_val,
+   Vector &q_der,
+   Vector &q_det,
+   Vector &q_nor,
+   const int eval_flags)
+{
+   MFEM_PERF_SCOPE("FaceQuadInterpolator::SmemEval3D");
```

Comment:
Should we keep this?

Reply:
It's a noop in general (as you know since you added the feature =]) and it helps with kernel naming in the profilers so I would personally like to keep it!

Reply:
I agree, we can start populating our kernels with these annotations - that should help.